src/Compatibility/Core/src/iOS/CollectionView/ItemsViewController.cs

```diff
@@ -19,7 +19,7 @@ public abstract class ItemsViewController<TItemsView> : UICollectionViewControll
 		public TItemsView ItemsView { get; }
 		protected ItemsViewLayout ItemsViewLayout { get; set; }
 		bool _initialized;
-		bool _isEmpty;
+		bool _isEmpty = true;
```

Comment:
Prevent CollectionView.EmptyView from showing up while it shouldn't on iOS
https://github.com/xamarin/Xamarin.Forms/pull/15122